libr/core/cmd_zign.c

```diff
@@ -174,84 +174,81 @@ static void addFcnZign(RCore *core, RAnalFunction *fcn, const char *name) {
 	}
 }
 
-static bool parseGraphMetrics(const char *args0, int nargs, RSignGraph *graph) {
-	const char *ptr = NULL;
-	int i = 0;
-
-	graph->cc = -1;
-	graph->nbbs = -1;
-	graph->edges = -1;
-	graph->ebbs = -1;
-	graph->bbsum = 0;
-
-	for (i = 0; i < nargs; i++) {
-		ptr = r_str_word_get0 (args0, i);
+static bool addCommentZign(RCore *core, const char *name, RList *args) {
+	if (r_list_length (args) != 1) {
+		eprintf ("Too many arguments for comment signature\n");
```

Comment:
"Invaild number of arguments" is better, because if args.len == 0 it wont make much sense to refer it